src/components/storefront/CartItem/CartItem.js

```diff
@@ -32,22 +38,24 @@ const CartItem = ({ title, sku, image, price, quantity, lineItemId, checkoutId }
     <Flex justifyContent="space-between">
       <Flex>
         <Flex direction="column">
-          <Image boxSize="200" objectFit="cover" border="1px solid black" src={image} alt={title} />
+          <Link as={PreserveQueryParamsLink} to={`products/${slug}`}>
+            <Image boxSize="200" objectFit="cover" border="1px solid black" src={image} alt={title} />
+          </Link>
         </Flex>
         <VStack alignItems="flex-start" w={64} pl={6}>
-          <Heading as="h4" size="subtitle" textTransform="uppercase" color="brand.gray">
+          <Heading size="lightCaption" textTransform="uppercase" color="brand.gray">
             {`Item #${sku}`}
           </Heading>
-          <Heading as="h4" size="subtitle" textTransform="uppercase" color="brand.darkgray">
+          <Heading size="subtitle" textTransform="uppercase" color="brand.darkgray">
```

Comment:
I think the "as" renders it as an `h4` HTML component, while `size` is trying to do the styilng. I would keep the `as`, and if it's breaking some of the styles, just pass them in as props here / below / above as needed.